traefik/values.yaml

```diff
@@ -120,6 +120,8 @@ ingressRoute:
     annotations: {}
     # Additional ingressRoute labels (e.g. for filtering IngressRoute by custom labels)
     labels: {}
+    # Specify the target containers for the dashboard ingress route, (e.g. traefik, web, websecure).
```

Comment:
```suggestion
+    # Specify the allowed entrypoints to use for the dashboard ingress route, (e.g. traefik, web, websecure).
+    # By default, it's using traefik entrypoint, which is not exposed.
+    # /!\ Do not expose your dashboard without any protection over the internet /!\
```

Reply:
👍🏼 thanks for revising and the security notice.
Applied.